packages/loaders/src/Progress.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { keyframes } from 'styled-components';
+
+const { zdColorBlue600, zdColorGrey200, zdSpacingXs } = require('@zendeskgarden/css-variables');
+
+const heights = {
+  small: 2,
+  large: 12,
+  default: 6
+};
+
+const sizeToHeight = size => heights[size || 'default'];
+const sizeToBorderRadius = size => sizeToHeight(size) / 2;
+
+const ProgressBarContainer = styled.div.attrs(props => ({
```

Comment:
Can you use the `<progress>` element? Or is it tricky to do custom styles on?

Reply:
We discussed that, but it is very hard to style as it maps to a native control in different OS'es.